docOps/docs/libraries/python/examples.md

```diff
@@ -0,0 +1,761 @@
+# Examples
+This section will guide you through several examples using the python binding of the `Wallet.rs` library. You can also find the code for the examples in the `/bindings/python/examples` folder in the [official GitHub repository](https://github.com/iotaledger/wallet.rs/tree/develop/bindings/python/examples).
+
+All the examples in this section expect your custom password  to be set in the  _.env_  file:
+```bash
+SH_PASSWORD="here is your super secure password"
+```
+
+## Security
+:::warning
+It is not recommended to store passwords on the host's environment variables, or in the source code in a production setup. 
+Please make sure you follow our [backup and security recommendations](https://chrysalis.docs.iota.org/guides/backup_security.html) for production use.
+:::
+
+
+## Account Manager and Individual Accounts
+You can initialize (open) a secure storage for individual accounts.  The storage is backed up by `Stronghold` by default, using an AccountManager instance.  
+
+The following example creates a new database and account:
+```python
+# Copyright 2020 IOTA Stiftung
+# SPDX-License-Identifier: Apache-2.0
+
+
+import iota_wallet as iw
+import os
+from dotenv import load_dotenv
+
+
+# Load the env variables
+load_dotenv()
+
+# Get the Stronghold password
+STRONGHOLD_PASSWORD = os.getenv('STRONGHOLD_PASSWORD')
+
+account_manager = iw.AccountManager(
+    storage_path='./alice-database'
+)  # note: `storage` and `storage_path` have to be declared together
+
+account_manager.set_stronghold_password(STRONGHOLD_PASSWORD)
+
+# mnemonic (seed) should be set only for new storage
+# once the storage has been initialized earlier then you should omit this step
+account_manager.store_mnemonic("Stronghold")
+```
+* Storage is initialized under the given path (`./alice-database`)
+* The password is set based on your password in  _.env_  file ( _manager.setStrongholdPassword(process.env.SH_PASSWORD)_ )
+* When you initialize the new database, a Stronghold mnemonic (seed) is automatically generated and stored by default ( _manager.storeMnemonic(SignerType.Stronghold)_ ).
+* The seed should be set only for the first time. In order to open already initialized database, you can simply use your password.
+
+The storage is encrypted at rest, so you need a strong password and location where to place your storage. 
+
+:::warning
+We highly recommended you store your `Stronghold` password encrypted on rest and separated from `Stronghold` snapshots. 
+
+Deal with the password with utmost care
+:::
+
+Technically speaking, the storage comprises two things:
```

Comment:
It's a dupe of nodejs description and isn't really an example. Maybe move it somewhere else?